router/router.go

```diff
@@ -156,7 +155,6 @@ func newExchangeMap(cfg *config.Configuration) map[string]adapters.Adapter {
 		"rubicon": rubicon.NewRubiconAdapter(adapters.DefaultHTTPAdapterConfig, cfg.Adapters[string(openrtb_ext.BidderRubicon)].Endpoint,
 			cfg.Adapters[string(openrtb_ext.BidderRubicon)].XAPI.Username, cfg.Adapters[string(openrtb_ext.BidderRubicon)].XAPI.Password, cfg.Adapters[string(openrtb_ext.BidderRubicon)].XAPI.Tracker),
 		"lifestreet": lifestreet.NewLifestreetAdapter(adapters.DefaultHTTPAdapterConfig, cfg.Adapters[string(openrtb_ext.BidderLifestreet)].Endpoint),
-		"conversant": conversant.NewConversantAdapter(adapters.DefaultHTTPAdapterConfig, cfg.Adapters[string(openrtb_ext.BidderConversant)].Endpoint),
```

Comment:
We're fine with this removal. Just want you to understand that publishers still using the legacy endpoint will now receive an invalid bidder error. If you want to continue with legacy endpoint support, you can still include the old adapter code and tests in legacy.go and legacy_test.go. We would remove those for you when legacy support is retired.

Reply:
We do want to keep the legacy support for now, so I'm going to add those back in today.

Reply:
I've added the legacy support back in under a separate file, but I still don't really follow how support for both is handled

Reply:
There are different interfaces and methods for the legacy endpoint versus the openrtb endpoint.

The legacy endpoint only works with adapters which specifically implement the legacy interface.

The openrtb endpoint has a wrapper to translate between the legacy model and the openrtb model, so it can use legacy adapters as is. That was meant as a transitionary approach. Now that we're looking to remove the legacy endpoint entirely, all adapters need to specifically implement the openrtb interface and not rely on the underlying conversion.

Does that explain it any clearer?

Reply:
yes, thank you